### ISSUE TITLE
Wrong ratio for recipes that produce multiple items 

### ISSUE THREAD

**Issue by monk-time** [Murnto/FactorioData#2]
I'm not sure this is the right place to post this, but there's no issue tracker on https://github.com/Murnto/factoratio, and other forks seem to be inactive.

Unless my calculations are wrong, the correct ratio for [copper cable](https://wiki.factorio.com/index.php?title=Copper_cable) on Assembling Machine 1 (crafting speed 0.5) should be 2 units/sec/factory ((2 units / 0.5 sec) \* 0.5) or **120 u/m/factory**, but the app shows only **60 u/m/factory**. I suspect it is caused by the recipe producing 2 items per input.

![Image](http://i.imgur.com/7DvrF3e.png)

Same with [alien science packs](https://wiki.factorio.com/index.php?title=Alien_science_pack): **25 units/m/factory** on Assembling Machine 1 [60 sec \* (10 units / 12 sec) \* 0.5], but the apps shows a speed that is 10 times smaller: **2.5 u/m/factory**.

![Image](http://i.imgur.com/BRGoenP.png)

So basically all ratios for items that produce multiple items per input are wrong.

**Comment by Murnto** [comment]
Should be fixed in Murnto/factoratio@a7b86c7 / 5702e37
